finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
Is there a way to make it a separate channel handler? I guess my question is can we "delay" channel init?